app/models/notification.rb

```diff
@@ -5,15 +5,10 @@ class Notification < ApplicationRecord
   has_many :users, through: :user_notifications
   belongs_to :generator_user, class_name: 'User', optional: true
 
-  enum type_of: Extends::NOTIFICATIONS_TYPES
-
-  def create_user_notification(user)
-    return if user == generator_user
-    return unless can_send_to_user?(user)
-    return unless user.enabled_notifications_for?(type_of.to_sym, :web)
+  include Noticed::Model
+  belongs_to :recipient, polymorphic: true
 
-    user_notifications.create!(user: user)
-  end
+  enum type_of: Extends::NOTIFICATIONS_TYPES
```

Comment:
If it is only needed for migration and deleted after, than move it to the migration file

Reply:
i removed and just used id for types